src/test/java/VerticalServiceImplTest.java

```diff
@@ -0,0 +1,109 @@
+import model.BlindVertical;
+import model.MountType;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import service.blindService.VerticalService;
+import service.blindService.VerticalServiceImpl;
+import service.catalog.ExchangeRate;
+import service.catalog.ExchangeRateImpl;
+import service.catalog.PriceCatalogVertical;
+import service.catalog.PriceCatalogVerticalImpl;
+import service.fileService.DataReader;
+import service.fileService.DataReaderImpl;
+import service.fileService.Property;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class VerticalServiceImplTest {
+    BlindVertical blindVertical;
+    DataReader dataReader = new DataReaderImpl();
+    Property property = new Property();
+    ExchangeRate exchangeRate = new ExchangeRateImpl(property);
+    PriceCatalogVertical priceCatalogVertical = new PriceCatalogVerticalImpl(dataReader, property);
+    VerticalService verticalService = new VerticalServiceImpl(blindVertical, priceCatalogVertical, exchangeRate);
```

Comment:
Can use something like @InjectMocks or @Spy instead of variable assignment?

Reply:
done